nx-X11/programs/Xserver/hw/nxagent/Keystroke.c

```diff
@@ -115,9 +122,13 @@ struct nxagentSpecialKeystrokeMap default_map[] = {
   {KEYSTROKE_IGNORE, ControlMask, True, XK_BackSpace},
   {KEYSTROKE_IGNORE, 0, False, XK_Terminate_Server},
   {KEYSTROKE_FORCE_SYNCHRONIZATION, ControlMask, True, XK_j},
+#ifdef DUMP
   {KEYSTROKE_REGIONS_ON_SCREEN, ControlMask, True, XK_a},
+#endif
+#ifdef NX_DEBUG_INPUT
   {KEYSTROKE_TEST_INPUT, ControlMask, True, XK_x},
   {KEYSTROKE_DEACTIVATE_INPUT_DEVICES_GRAB, ControlMask, True, XK_y},
+#endif
```

Comment:
What happens, if keystrokes.cfg defines strokes for KEYSTROKE_TEST_INPUT,  KEYSTROKE_DEACTIVATE_INPUT_DEVICES_GRAB,  and KEYSTROKE_DEBUG_TREE but the support has not been built in.

Are those keystrokes.cfg parameters then safely ignored? Did you test that?